packages/dropdownMenu/tests/DropdownMenu.test.tsx

```diff
@@ -150,16 +151,19 @@ describe("Dropdown", () => {
       </DropdownMenu>
     );
 
-    const trigger = component.find(`#${triggerId}`);
-    expect(component.find(DropdownContents).length).toBe(0);
-    trigger.simulate("click");
-    expect(component.find(DropdownContents).prop("isOpen")).toBe(true);
-    trigger.simulate("click");
-    expect(component.find(DropdownContents).length).toBe(0);
+    const trigger = screen.getByText(/dropdown trigger/i);
+
+    expect(screen.queryByText(/edit/i)).toBeFalsy();
```

Comment:
Modifed logic due to the differences in testing approaches between Enzyme and RTL. Grab this component programmatically via a CSS selector vs grab "what the user sees".